src/Controller/Component/CrudComponent.php

```diff
@@ -547,7 +547,7 @@ public function defaults($type, $name, $config = null)
                 $this->config(sprintf('%s.%s', $type, $realName), $config);
             }
 
-            return;
+            return null;
```

Comment:
Even if it is not a democracy: ***If*** it doesn't break anything I am all for explicitly returning null as returning an object/data OR void is strange. Used as a setter in future it would be best to just return $this.

However is there is only a slight chance of breaking existing user-land code I am against it.